code/modules/mob/living/carbon/xenomorph/castes/queen/queen.dm

```diff
@@ -100,6 +100,12 @@
 	playsound(loc, 'sound/voice/alien_queen_died.ogg', 75, 0)
 
 /mob/living/carbon/xenomorph/queen/xeno_death_alert()
+	if(!isdistressgamemode(SSticker.mode))
+		return;
```

Comment:
rogue ;
and you're istyping all the time, there is a flag for psypoint use, that's what you should check

Reply:
Oh fuck, my c# accidentially carried over. Done.